packages/superset-ui-chart-composition/package.json

```diff
@@ -26,7 +26,7 @@
     "access": "public"
   },
   "dependencies": {
-    "@types/react": "^16.9.34",
+    "@types/react": "^16.9.35",
```

Comment:
I'm updating `@types/react` everywhere as there's some related typing errors while `npm link` in Superset app. Don't want to dig out which version is causing the issue.

Reply:
Please do this in a separate PR. It is easier to review + smoke test + separate change if revert is needed. Can also couple with the demo script in `package.json` / storybook config. I can stamp that one quickly.

Reply:
The storybook change is [already in `master`](https://github.com/apache-superset/superset-ui/pull/600) (thought it was safe so I merged it myself). I just need to update the feature branch. Will also create a PR to upgrade react types.